src/pages/edge/adobe-journey-optimizer/code-based/tutorial.md

```diff
@@ -118,3 +119,17 @@ Android
 iOS
 
 <Tabs query="platform=ios&function=track-with-tokens"/>
+
+## Tracking interactions with content cards
+
+Since the onus of rendering the content card lies with the app developer, you must monitor the display and desired end user interactions and call the appropriate tracking APIs. To record a display or interaction with the content card, call the [track](../public-classes/content-card.md#track) API provided in the `ContentCard` class. The following code shows two examples of tracking: when the content card is displayed to the user and when the user clicks on the content card. These examples are for illustrating how to call the track API and not a recommendation on where it should be called. Please examine your app workflow to find the appropriate way to perform tracking.
```

Comment:
I feel like "the onus of..." is a little bit too big language-y for my liking. I'd just go with "Since rendering the content card lies..."

Reply:
Hehe chatgpt suggestion gone too far, will fix it 😄